src/applications/disability-benefits/686c-674/config/chapters/veteran-information/veteran-information/VeteranInformationComponent.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+const VeteranInformationComponent = ({
+  user: {
+    gender,
+    dob,
+    userFullName: { first, last },
+  },
+}) => {
+  let dateOfBirthFormatted = '-';
+  let genderFull = '-';
+  if (dob) {
+    dateOfBirthFormatted = moment(dob).format('MMMM Do YYYY');
+  }
+  if (gender === 'M') {
+    genderFull = 'Male';
+  } else if (gender === 'F') {
+    genderFull = 'Female';
+  }
+  const alertContent = (
+    <dl className="vads-u-margin--0 vads-u-padding-left--2">
+      <dt className="vads-u-line-height--4 vads-u-padding-bottom--2">
+        <strong>
+          {first} {last}
+        </strong>
+      </dt>
+      <dd className="vads-u-line-height--4 vads-u-padding-bottom--2">
+        Date of birth: {dateOfBirthFormatted}
+      </dd>
+      <dd className="vads-u-line-height--4">Gender: {genderFull}</dd>
+    </dl>
+  );
+  return (
+    <div>
+      <p>This is the information we have on file for you.</p>
```

Comment:
I think this should read 'personal information' instead of just 'information'. [Mockups](https://xd.adobe.com/view/ca825c11-2268-458f-4fec-6aaecfec5b2d-bcb3/screen/dc153448-40b1-4d35-b890-698dca690305/Add-Modify-Dependents-Veteran-Details)

Reply:
Nice catch! updated